src/plots/plots.js

```diff
@@ -41,7 +41,7 @@ plots.fontWeight = 'normal';
  */
 plots.register = function(_module, thisType, categoriesIn, meta) {
     if(modules[thisType]) {
-        console.warn('type ' + thisType + ' already registered');
+        console.log('type ' + thisType + ' already registered');
```

Comment:
Fyi I switched log --> warn out of (perhaps bad) habit, from when I had a project that needed to support ie8 :frowning: